pkg/provisioner/k8sprovisioner.go

```diff
@@ -16,7 +17,7 @@ import (
 )
 
 type k8sProvisioner struct {
-	Namespace              string
+	Config                 config.ProvisionerWorkerConfig
 	Clientset              *kubernetes.Clientset
 	Pods                   corev1.PodInterface
```

Comment:
No real need for these to be exported. Could you "unexport" all these struct fields?